### ISSUE TITLE
Wrong data

### ISSUE THREAD

**Issue by pronoua** [lrybak/hassio-owserver#8]
I have random data from the sensor.
The sensor is DS18B20
![image](https://user-images.githubusercontent.com/44452247/186756479-c4b54843-d7a7-4a06-aed2-4de88215a171.png)

**Comment by pronoua** [comment]
![image](https://user-images.githubusercontent.com/44452247/186756822-7f395b14-59cc-4206-b239-47c95d3d8546.png)

**Comment by pronoua** [comment]
From w1-bus the data is true:
![image](https://user-images.githubusercontent.com/44452247/186761362-188e6c71-1b8f-42f0-ae8e-2cd941bbc125.png)

But in owserver its...random? I gues..
![image](https://user-images.githubusercontent.com/44452247/186761969-2cf85331-32c6-43e1-b3f0-aa1c144191f2.png)

**Comment by pronoua** [comment]
Anybody... ?

**Comment by AllardKatan** [comment]
I have the exact same problem as pronoua, using the ds18b20 temperature sensor on a Pi 3 B. 
Pity, I was quite happy to find a way to use a sensor directly on the Pi that runs my HomeAssitant. Will have to come up with a different solution now, unless somebody knows a way to fix this...

**Comment by lrybak** [comment]
Sorry for late response.
I'm affraid my addon has nothing to do with your issue. It simply runs owserver which is then pulled by HomeAssistant.
Can you let me know about your setup, especially how you wire your ds18b20 sensors?

Addon is intended to be used with 1wire sensors connected to owserver through the bus master serial devices based on DS2480B chip. 
Whenever you will not select your USB device in addon configuration, **addon will start in fake mode** showing random sensors with random readings.

**Comment by AllardKatan** [comment]
Hi, thanks for your reply. I have a switch and a ds18b20 connected to gpio 4 and 27 on the Pi. I just found out an hour ago that indeed the owserver was running in fake mode. Which happens to simulate a switch and a ds18b20...

I can't really find how to integrate this setup with Home Assistant. It seems like Hass is mostly built to use the RPi as the hub, using the gpio to mount sensors and actuators is quite cumbersome. I just started this project so maybe I will find a better way....
If I understand correctly, one solution would be to get a serial to 1 wire interface like this https://www.artekit.eu/products/breakout-boards/io/ak-ds2480b-uart/ and use your add-on?
